src/panels/config/zha/zha-groups-dashboard.ts

```diff
@@ -48,8 +73,31 @@ export class ZHAGroupsDashboard extends LitElement {
     this._groups = (await fetchGroups(this.hass!)).sort(sortZHAGroups);
   }
 
+  private _handleRemoveSelectionChanged(ev: CustomEvent): void {
+    const changedSelection = ev.detail as SelectionChangedEvent;
+    const groupId = Number(changedSelection.id);
+    if (changedSelection.selected) {
+      this._selectedGroupsToRemove.push(groupId);
+    } else {
+      const index = this._selectedGroupsToRemove.indexOf(groupId);
+      if (index !== -1) {
+        this._selectedGroupsToRemove.splice(index, 1);
+      }
+    }
+    this._canRemove = this._selectedGroupsToRemove.length > 0;
+  }
+
+  private async _removeGroup(): Promise<void> {
+    this._processingRemove = true;
+    this._groups = await removeGroups(this.hass, this._selectedGroupsToRemove);
+    this._selectedGroupsToRemove = [];
+    this._canRemove = false;
+    this._processingRemove = false;
+  }
+
   static get styles(): CSSResult[] {
     return [
+      haStyleDialog,
```

Comment:
It is not a dialog?

Reply:
If I’m not crazy... I am using a style from this on the remove button. At least that’s what I intended :)

Reply:
I think it is better to just extract that one style, this is really for styling dialogs